code/_helpers/unsorted.dm

```diff
@@ -237,7 +237,7 @@ Turf and target are seperate in case you want to teleport some distance from a t
 			line+=locate(px,py,M.z)
 	return line
 
-#define LOCATE_COORDS(X, Y, Z) locate(between(1, X, world.maxx), between(1, Y, world.maxy), Z)
+#define LOCATE_COORDS(X, Y, Z) locate(Clamp(X, 1, world.maxx), Clamp(Y, 1, world.maxy), Z)
```

Comment:
clamp() is default Byond expression, use it instead.
```suggestion
#define LOCATE_COORDS(X, Y, Z) locate(clamp(X, 1, world.maxx), clamp(Y, 1, world.maxy), Z)
```

Reply:
> clamp() is default Byond expression, use it instead.

goddamn it, welp time to regex search and replace